lib/portlayer/exec/container.go

```diff
@@ -596,6 +596,8 @@ func (c *Container) Remove(ctx context.Context, sess *session.Session) error {
 				log.Errorf("Error while attempting to unregister container VM: %s", err)
 				return err
 			}
+		case *types.ConcurrentAccess:
```

Comment:
I need to document those quirks, without llpm we are getting occasional ConcurrentAccess errors during DeleteExceptDisk, that's why we are ignoring this